lib/src/hashing.rs

```diff
@@ -43,8 +44,9 @@ impl Hasher for Sha256 {
 pub struct Blake2b;
 
 impl Hasher for Blake2b {
-    fn calculate_digest(&self, sg: &SGData) -> Vec<u8> {
-        let mut blake2 = blake2::Blake2b::default();
+    fn calculate_digest(&self, sg: &SGData) -> Vec<u8>
+    {
```

Comment:
Since I'm already pointing out other stuff: `{` on the prev line. I'd recommend setting up `rustfmt` on save or something to avoid worrying about it.

Reply:
applied